test/server/cards/03_TWI/units/VanguardDroidBomber.spec.ts

```diff
@@ -0,0 +1,46 @@
+describe('Vanguard Droid Bomber', function () {
+    integration(function (contextRef) {
+        describe('Vanguard Droid Bomber\'s ability', function () {
+            it('should deal 2 domage to the opponent base', function () {
```

Comment:
nit: just to help other people understand what this test is doing (in case they break it on accident), we should include the reason it's happening to differentiate it from the other test and don't know the card text. So like 'should deal 2 damage to the opponent base because there is a friendly separatist unit'. Same for the other test title